modules/apps/portal-security/portal-security-ldap-api/src/main/java/com/liferay/portal/security/ldap/configuration/BaseConfigurationProvider.java

```diff
@@ -46,9 +47,7 @@ protected String getMetatypeId() {
 		return _factoryPid;
 	}
 
-	protected abstract void setConfigurationAdmin(
-		ConfigurationAdmin configurationAdmin);
-
+	@Reference
```

Comment:
@W2jx The change here is weird, the current class is not a component, but you are using @Reference.

Tina.